src/easydynamics/sample_model/components/gaussian.py

```diff
@@ -19,7 +17,7 @@ class Gaussian(CreateParametersMixin, ModelComponent):
     If the center is not provided, it will be centered at 0 and fixed, which is typically what you want in QENS.
 
```

Comment:
your component functions are missing `get_all_variables` Implementation, relying on the parent class implementation.
The parent `ModelBase.get_all_variables()` may not find _area, _center, _width since they're not passed to super().init() anymore.